client/app/states/administration/rules/rules-list.directive.js

```diff
@@ -0,0 +1,368 @@
+/* eslint camelcase: "off" */
+(function() {
+  'use strict';
+
+  angular.module('app.states')
```

Comment:
given this is a directive it probably should be in the components folder and attached to `app.components`